docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -458,6 +398,8 @@ sudo python3 plant-watering-robot.py
 ```
 
 To tinker this example code to work best for you, determine at what [analog value from the soil moisture readings](#test-your-soil-moisture-readings-on-your-pi) you want to water your plant, as your thirsty plant's average moisture reading might differ from our example value of `60000`.
```

Comment:
```suggestion
```

Now that we've removed the adafruit CLI section, I think it's safe to just delete this line entirely: we no longer provide users with guidance on adjusting this value.